core/src/avm1/globals/text_field.rs

```diff
@@ -138,10 +138,27 @@ pub fn create_proto<'gc>(
         "type" => [get_type, set_type],
         "variable" => [variable, set_variable],
         "wordWrap" => [word_wrap, set_word_wrap],
+        "password" => [is_password, set_password],
     );
 
     object.into()
 }
+pub fn is_password<'gc>(
```

Comment:
Let's name this specific function `password` in this specific spot  to match the Flash property name `password`, and to match the naming convention we use for the other Flash methods (`property` & `set_property`).